apiconcepts/projectautomation/working_with_project_reports.md

```diff
@@ -0,0 +1,113 @@
+Working with Project Reports
+==
+
+In Trados Studio, the project files have associated a set of predefined reports. The built-in reports are visible in the Reports Viewer. Trados Studio Reports API allows the 3rd party developers to extend and customize the existing set of reports and display them within an additional Trados Studio Viewer using a custom developed plugin. 
```

Comment:
instead of Trados Studio use <Var:ProductName> so that we can easily change the product name